tornado/ioloop.py

```diff
@@ -276,22 +263,36 @@ def current(instance=True):
            Added ``instance`` argument to control the fallback to
            `IOLoop.instance()`.
         .. versionchanged:: 5.0
+           On Python 3, control of the current `IOLoop` is delegated
+           `asyncio`, with this and other methods pass-through accessors.
```

Comment:
grammar nit:
"is delegated *to* asyncio"
"this and other methods *as* pass-through accessors" (or maybe *being*)